pkgs/test/pubspec.yaml

```diff
@@ -33,8 +33,8 @@ dependencies:
   web_socket_channel: '^1.0.0'
   yaml: '^2.0.0'
   # Use an exact version until the test_api and test_core package are stable.
-  test_api: '0.2.2'
-  test_core: '0.2.1+1'
+  test_api: '0.2.3'
```

Comment:
Missed version bump for `pkg:test` – going to do that in a separate PR?

Reply:
Was an oversight. Fixed now and merged the changelog entry from the unpublished version.